tests/test_pedigree.py

```diff
@@ -174,3 +175,153 @@ def test_pedigree_sanity_checks(self):
 
         with pytest.raises(ValueError):
             ped.get_times(individual=ped.individual)
+
+
+def simulate_pedigree(
+    n_founders=8,
+    n_children_prob=None,
+    n_generations=3,
+    random_sex=True,
+) -> tskit.IndividualTable:
```

Comment:
Also need some documentation here describing what the function does. It doesn't need to  be the full sphinx approach we'd use for a public function that's part of the API, but developers should be able to read a quick summary to understand what the function does and what the parameters mean.